streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -745,15 +743,22 @@ private void runLoop() {
             try {
                 runOnce();
                 if (assignmentErrorCode.get() == AssignorError.VERSION_PROBING.code()) {
-                    log.info("Version probing detected. Triggering new rebalance.");
+                    log.info("Version probing detected. Rejoin the consumer group to trigger a new rebalance.");
```

Comment:
```suggestion
                    log.info("Version probing detected. Rejoining the consumer group to trigger a new rebalance.");
```

It sounded like you were telling the operator to do something, but you're just telling them that something is going to happen.